.ddev/commands/web/gulp

```diff
@@ -1,7 +1,32 @@
 #!/bin/bash
 
+set -e
+
 ## Description: Run robo commands inside the web container for the client app.
 ## Usage: gulp
 ## Example: "ddev gulp publish" or "ddev gulp".
 
+CLIENT_PATH="../../client"
+
+if [ "$1" == "publish" ];
+then
+  if [[ -z "${EHEZA_INFRA_REPO_REMOTE}" ]]; then
+    echo "Please specify EHEZA_INFRA_REPO_REMOTE in your DDEV local config before trying to do deployments."
+    exit 1
+  fi
+
+  if [ ! -d ../../eheza-infra ]; then
+  echo "Cloning eheza-infra repo..."
+    git clone "${EHEZA_INFRA_REPO_REMOTE}" ../../eheza-infra
+  else
+    echo "Updating eheza-infra repo..."
+    cd ../../eheza-infra
+    git pull
+    cd -
+  fi
+
+  cp ../../eheza-infra/elm/Config.Deploy.elm ${CLIENT_PATH}/src/elm/
```

Comment:
This means that we need to have that `Config.Deploy.elm` under `elm` folder.
Maybe it should be under `config`, or maybe even under root?

Reply:
https://github.com/Gizra/ihangane-infra/pull/11/files - as it's multi-purpose repository, I'd not put it under root.
With `elm`, we distinguish it from backend-ish things.